src/locales/de_DE.json

```diff
@@ -231,197 +253,231 @@
 "label.action.stop.systemvm.processing": "System-VM wird gestoppt....",
 "label.action.take.snapshot": "Schnappschuss erstellen",
 "label.action.take.snapshot.processing": "Schnappschuss wird gemacht....",
-"label.action.unmanage.cluster": "Vernachl\u00e4ssige Cluster",
+"label.action.template.permission": "Update Template Erlaubnis",
+"label.action.template.share": "Update Template Verteilung",
+"label.action.unmanage.cluster": "Vernachlässige Cluster",
 "label.action.unmanage.cluster.processing": "Cluster wird nicht mehr verwaltet....",
-"label.action.update.os.preference": "Betriebssystem Pr\u00e4verenz aktualisieren",
+"label.action.unmanage.virtualmachine": "Verwaltung der VM beenden",
+"label.action.update.offering.access": "Update Zugangsangebot",
```

Comment:
Access-Offering aktualisieren

Reply:
@onitake i dont would use "Access", this is not german. we need an better replace. some ideas here?

Reply:
@utchoang lets us go with "Berechtigungsangebot aktualisieren"